docs/api/bb-api.md

```diff
@@ -0,0 +1,36 @@
+# Vets.gov Health Records API
+
+### API for Blue Button info from My HealtheVet
+
+ Allows for generating and downloading a veteran's health records
+
+##### Available routes
```

Comment:
Could you add a note similar to what we discussed in the endpoint PR I reviewed, about the idempotency/asynchonousness (?) of these API endpoints (e.g. possibility of delay between calling POST /health_records and /GET health_records). I think it would be helpful for future reference.